### ISSUE TITLE
[Payjoin Receive] initReceiver Error while creating receive URI

### ISSUE THREAD

**Issue by i5hi** [SatoshiPortal/bullbitcoin-mobile#370]
initReceiver throws the following error when creating a Receive uri

```
_Exception (Exception: Error initializing payjoin Receiver: PayjoinException( Error while parsing the address: legacy address base58 string ))
```

**Comment by i5hi** [comment]
Note: This does not prevent the address from being created and displayed.

**Comment by i5hi** [comment]
`legacy address base58 string`

The address being used is bech32 segwit

**Comment by i5hi** [comment]
Seems like its using the liquid address.

We should not use the `Instant Wallet` for payjoins.

**Comment by kumulynja** [comment]
@i5hi as mentioned in the PR that is already merged, maybe adding an extra check on the wallet type is also needed still instead of only the payment network, what do you think? Shall I add it before closing this issue completely?

**Comment by ethicnology** [comment]
Closed by #381
